cmd/testgrid-config-generator/README.md

```diff
@@ -47,7 +47,7 @@ $ git status
 modified:   groups.yaml
 new file:   redhat-openshift-...
 ```
-Commit the  changes and file a PR in https://github.com/kubernetes/test-infra/ to land them.
+PRs would be generated automatically for runs of the testgrid-config-generator tool which result in changes to `github.com/kubernetes/test-infra`. This is done by the periodic-prow-auto-testgrid-generator job.
```

Comment:
you should put this earlier.  no one needs to build+run testgrid-config-generator at all anymore.  You can leave the instructions as they are useful if someone needs to run it manually for some reason, but make it clear they normally don't need to do any of this any more.